confidential-ledger-demo/server/sampledata.json

```diff
@@ -1,9 +1,9 @@
 {
```

Comment:
Possibly worth moving this into the docs @FergusKidd

Reply:
Good idea, will include that in the documentation issue

Reply:
Perfect, I think that will suppress the security error in the scan